moto/ec2/models/instances.py

```diff
@@ -73,6 +73,27 @@ def __init__(self, options: Optional[dict[str, Any]] = None):
         self.instance_metadata_tags = options.get("InstanceMetadataTags", "disabled")
 
 
+class BlockDeviceWrapper:
```

Comment:
Can we use a regular method here instead? That would be easier to read IMO, and would achieve the same thing.

Something like:
```python
def get_block_status(volume_status):
    if volume_status == "in-use":
        return "attached"
    ...
```